and-bible/app/src/main/res/layout/list_item_3_highlighted.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<net.bible.android.view.util.widget.TwoLine2TitleListItem xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:minHeight="60dp"
+    android:mode="twoLine"
+    android:paddingLeft="16dp"
+    android:paddingRight="16dp"
+    android:background="@drawable/selectable_background">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical" >
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="8dp"
+            android:orientation="horizontal" >
+
+            <TextView android:id="@+id/text1"
```

Comment:
I was just adapting the original class in the code which also was like this.

 I can do this and the other suggestions to cleanup so that all of this fit to the style that you prefer, but: I only want to go through that effort if you do plan to incorporate this feature. If not it is not worth it. So bottom line: Do you intend to use this if cleaned up?